app/models/site_configuration.rb

```diff
@@ -67,6 +69,10 @@ def max_revision_reasons
     end
   end
 
+  def inbox_enabled_changed?
+    saved_change_to_inbox_enabled?
```

Comment:
is there a good reason to have a non-standard name for this boolean in addition to the standard one, thus making the codebase one step more crufted up?